File: DS Assignments/ProgrammingAssignment1/ProgrammingAssignment1/src/programmingassignment1/Nurse.java
// COSC 2320 Summer 2015
// Ali Elsaadi
// Progamming Assignment 1
// This is my own work; I will not post
package programmingassignment1;


public class Nurse extends HospitalEmployee{
    private int numPatients;
    
    public Nurse ()
    {
        super (" ", 0);
        numPatients = 0;
    }
    
    public Nurse (String empName, int empNumber, int numPat)
    {
        super (empName, empNumber);
        numPatients = numPat;
    }
    
    public void setNumPatients (int pat)
    {
        numPatients = pat;
        
    }
    
    public int getNumPatients()
    {
        return numPatients;
    }
    
    public String toString()
    {
        return super.toString() + "Number of patients: " + numPatients;
    }
}

File: DS Assignments/ProgrammingAssignment4/ProgrammingAssignment4/ProgrammingAssignment4/Nurse.cpp
#include "Nurse.h"

Nurse::Nurse()
{
	Nurse::HospitalEmployee(" ", 0);
	numPatients = 0;
}
Nurse::Nurse(string empName, int empNumber, int numPat)
{
	Nurse::HospitalEmployee(empName, empNumber);
	numPatients = numPat;
}
void Nurse::setNumPatients(int pat)
{
	numPatients = pat;
}	
int Nurse::getNumPatients()
{
	return numPatients;
}
string Nurse::toString()
{
	string nbp;
	stringstream ss;
	ss << numPatients;
	nbp = ss.str();
	return Nurse::HospitalEmployee::toString() + "Number of patients: " + nbp;
}